modules/ossm-rn-new-features.adoc

```diff
@@ -17,13 +17,121 @@ This release adds improvements related to the following components and concepts.
 
 == New features {SMProductName} version {SMProductVersion}
 
-This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), bug fixes, and is supported on OpenShift Container Platform 4.9 or later.
+This release of {SMProductName} introduces new features, addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on OpenShift Container Platform 4.9, 4.10, and 4.11.
 
 === Component versions included in {SMProductName} version {SMProductVersion}
 
 |===
 |Component |Version
 
+|Istio
+|1.14
+
+|Envoy Proxy
+|1.22.4
+
+|Jaeger
+|1.38
+
+|Kiali
+|1.57.3
+|===
+
+=== New Container Network Interface (CNI) DaemonSet container and ConfigMap
+
+The `openshift-operators` namespace includes a new istio CNI DaemonSet `istio-cni-node-v2-3` and a new `ConfigMap`, `istio-cni-config-v2-3`.
+
+When upgrading to Service Mesh Control Plane 2.3, the existing `istio-cni-node` DaemonSet is not changed, and a new `istio-cni-node-v2-3` DaemonSet is created.
+
+This name change does not affect previous releases or any `istio-cni-node` CNI DaemonSet associated with a Service Mesh Control Plane deployed using a previous release.
+
+=== Gateway injection support
+
+This release introduces generally available support for Service Mesh Gateway injection. Gateway configurations are applied to standalone Envoy proxies that are running at the edge of the mesh, rather than the sidecar Envoy proxies running alongside your service workloads. This enables the ability to customize gateway options. When using gateway injection, you must create the following resources in the namespace where you want to run your gateway proxy: `Service`, `Deployment`, `Role`, and `RoleBinding`.
```

Comment:
@rh-tokeefe I think that @longmuir meant to remove "Service Mesh" from the first sentence. I also suggest to remove it, because Gateway injection is a pure upstream feature, so "Service Mesh" prefix is ambiguous and may suggest that it's something OSSM-specific (what may be not appealing for users).
```suggestion
This release introduces generally available support for Gateway injection. Gateway configurations are applied to standalone Envoy proxies that are running at the edge of the mesh, rather than the sidecar Envoy proxies running alongside your service workloads. This enables the ability to customize gateway options. When using gateway injection, you must create the following resources in the namespace where you want to run your gateway proxy: `Service`, `Deployment`, `Role`, and `RoleBinding`.
```

Reply:
Thanks @jewertow the update has been made.